execution/src/windows/execution.cc

```diff
@@ -74,6 +74,10 @@ namespace leatherman { namespace execution {
 
     string which(string const& file, vector<string> const& directories, bool expand)
     {
+        //On Windows this function is not supported
+        if (!expand) {
+            throw std::invalid_argument("Unsupported argument on Windows");
```

Comment:
Leatherman has some defined exceptions that wrap the standard library ones.
Looks like the standard in this file is to throw `execution_exception`.

Also, maybe we can log the message using `LOG_ERROR`, like the below cases.

Reply:
LOG_ERROR -agree
execution_exception should be used for "normal" execution that encounter problems.
std::invalid_argument shows that it is not "normal" execution (on windows )